requirements/dev.txt

```diff
@@ -208,7 +208,7 @@ pylint-plugin-utils==0.6
     #   -r requirements/quality.txt
     #   pylint-celery
     #   pylint-django
-pyparsing==3.0.3
+pyparsing==2.4.7
```

Comment:
Version is getting downgraded due to an internal constraint of packaging which will be resolved in the next version release.